src/java.desktop/macosx/classes/com/apple/laf/AquaComboBoxUI.java

```diff
@@ -133,7 +139,13 @@ protected ItemListener createItemListener() {
         return new ItemListener() {
             long lastBlink = 0L;
             public void itemStateChanged(final ItemEvent e) {
+
                 if (e.getStateChange() != ItemEvent.SELECTED) return;
```

Comment:
```suggestion
            public void itemStateChanged(final ItemEvent e) {
                if (e.getStateChange() != ItemEvent.SELECTED) return;
```
The blank line at the start of the method isn't necessary, is it?

Reply:
Removed.